app/uploaders/image_uploader.rb

```diff
@@ -36,7 +36,7 @@ def store_dir
   end
 
   version :xlarge do
-    process resize_to_limit: [2500, 1200]
+    process resize_to_limit: [2500, 1900]
```

Comment:
I think we'd want a new version if you're going to change the resize limitation, otherwise this could affect the intention of xlarge for other kiosks?

Reply:
Also, should it be resized to 1200x1900 somehow?

Reply:
Ill call it, :xlong and I can easily make it 1200 by 1900, but ill just have to see where we make the choice to display a specific image size.